system/modules/core/forms/FormTextArea.php

```diff
@@ -144,7 +144,7 @@ public function __get($strKey)
 				break;
 
 			case 'value':
-				return specialchars(str_replace('\n', "\n", $this->varValue));
+				return str_replace('\n', "\n", $this->varValue);
```

Comment:
In [form_textarea.html5:20](https://github.com/contao/core/blob/3e273a87f35ebb3ecafa36d695c3e380a1712b26/system/modules/core/templates/forms/form_textarea.html5#L20) the value gets printed without escaping `<?= $this->value ?>`, so this change introduces a security issue I think.

If you fill out a form with `</textarea>` in a textarea field and don’t fill out all required fields, you can break out of the textarea and inject XSS code.